Ensure that extensions are marked as compatible with gnome-shell-3.2
(meaning 3.2.*), not just 3.2.0.

diff --git a/extensions/alternate-tab/metadata.json.in b/extensions/alternate-tab/metadata.json.in
index d811b54..8aa86e3 100644
--- a/extensions/alternate-tab/metadata.json.in
+++ b/extensions/alternate-tab/metadata.json.in
@@ -3,7 +3,7 @@
 "name": "AlternateTab",
 "description": "A replacement for Alt-Tab, allows to cycle between windows and does not group by application",
 "original-authors": [  "jw@bargsten.org", "thomas.bouffon@gmail.com" ],
-"shell-version": [ "@shell_current@" ],
+"shell-version": [ "@shell_current@", "3.2" ],
 "localedir": "@LOCALEDIR@",
 "url": "@url@"
 }
diff --git a/extensions/dock/metadata.json.in b/extensions/dock/metadata.json.in
index dce4a88..96237b4 100644
--- a/extensions/dock/metadata.json.in
+++ b/extensions/dock/metadata.json.in
@@ -3,7 +3,7 @@
 "name": "Dock",
 "description": "A dock for the GNOME Shell -- displays favorite and running applications",
 "original-author": "tclaesson@gmail.com",
-"shell-version": [ "@shell_current@" ],
+"shell-version": [ "@shell_current@", "3.2" ],
 "localedir": "@LOCALEDIR@",
 "url": "@url@"
 }
